src/Microsoft.TestPlatform.Execution.Shared/Microsoft.TestPlatform.Execution.Shared.projitems

```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MSBuildAllProjects Condition="'$(MSBuildVersion)' == '' Or '$(MSBuildVersion)' &lt; '16.0'">$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <HasSharedItems>true</HasSharedItems>
+    <SharedGUID>7f26eda3-c8c4-4b7f-a9b6-d278c2f40a13</SharedGUID>
+  </PropertyGroup>
+  <PropertyGroup Label="Configuration">
+    <Import_RootNamespace> Microsoft.VisualStudio.TestPlatform.Execution</Import_RootNamespace>
```

Comment:
```suggestion
    <Import_RootNamespace>Microsoft.VisualStudio.TestPlatform.Execution</Import_RootNamespace>
```

nit.

Reply:
Are we using this property anywhere?

Reply:
It is used to define the default namespace. otherwise the namespace would be the same as the name of the project.